document OIDC connector 'max_age' field (#31800)
gravitational/teleport: docs/pages/access-controls/sso/oidc.mdx

       roles: [ "access" ]
```

> ### Optional: Max Age

Reviewer: ```suggestion
### Optional: Max age
```
We use sentence case for headings

> Teleport supports setting the `max_age` field since version 13.3.7 to control the

Reviewer: ```suggestion
Teleport has supported setting the `max_age` field since version 13.3.7 to control the
```
To agree grammatically with "since version...".

> sessions before they will be forced to reauthenticate. By default `max_age` is empty,

Reviewer 2: ```suggestion
sessions before they will be forced to reauthenticate. By default `max_age` is unset,
```

default `max_age` is unset, meaning once a user authenticates using OIDC they will
not have to reauthenticate unless the configured OIDC provider forces them to. This
can be set to a duration of time to force users to reauthenticate more often. If

> `max_age` is set to zero seconds, users will be forced to reauthenticate every time.

Reviewer: I think "forced to reauthenticate every time" can be more precise. Does it mean, "forced to reauthenticate with their OIDC provider every time they authenticate with Teleport"?

Author: That's correct, thanks

Note that the specified duration must be in whole seconds. `24h` works because that's
the same as `1440s`, but `60s500ms` would not be allowed as that is 60.5 seconds.

```yaml
# Extra parts of OIDC yaml have been removed.
spec:
  max_age: 24h
```

> Note that not all OIDC providers support setting `max_age`. Google and Gitlab are

Reviewer 2: ```suggestion
Note that not all OIDC providers support setting `max_age`. Google and GitLab are
```
